lib/main.dart

```diff
@@ -35,17 +34,44 @@ class MyHomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<MyHomePage> {
-
   @override
   Widget build(BuildContext context) {
     final _counter = Provider.of<Counter>(context);
 
     return Scaffold(
-      
       body: Center(
         child: Column(
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
+            Container(
```

Comment:
Wrap the container in a card so that you can apply elevation (I'd suggest 5 as it's subtle), make sure to add a shape to the card to of CircleBorder() so that it wraps the container

Reply:
On the container itself add a padding with EdgeInsets.all(3), what that will do is make is so that the white background color of the container is visible thereby acting as a border.

Reply:
![IconLogo class](https://user-images.githubusercontent.com/20342200/67843305-a1cbdd80-faf3-11e9-9411-fa5b356572d3.PNG)

Done, it doesn't sort out everything though.
The red color is for testing purpose only, so is the green attribute in the code.

![Screenshot last](https://user-images.githubusercontent.com/20342200/67843758-86ad9d80-faf4-11e9-9755-6543be12de2a.png)

Reply:
... And I have no clue where the black circle is coming from. I fiddled a bit to try to find out, without success.